resources/reacts/baltics.json

```diff
@@ -0,0 +1,85 @@
+{
+  "criteria": [
+    {
+      "match_link_id": "baltics_lt",
+      "match_whole_word": true,
+      "keywords": [
+        "Taivan(?:as|a|e|o|ui|ie(?:tiškas|tis|tė|či(?:ų|u|ai|ui|ams)))"
+      ]
+    },
+    {
+      "match_link_id": "baltics_lt",
+      "match_whole_word": false,
+      "keywords": [
+        "立陶宛"
+      ]
+    },
+    {
+      "match_link_id": "baltics_lv",
+      "match_whole_word": true,
+      "keywords": [
+        "Taivān(?:a|ā|ai|as|u|isks|ie(?:tis|te|šu|tim|tei|šiem|ti|te|ši|tes|ši))"
+      ]
+    },
+    {
+      "match_link_id": "baltics_lv",
+      "match_whole_word": false,
+      "keywords": [
+        "拉脫維亞"
+      ]
+    },
+    {
+      "match_link_id": "baltics_ee",
+      "match_whole_word": true,
+      "keywords": [
+        "Taiwan(?:l(?:ane|as(?:e|t|te(?:sse|s|st|le|l|lt|ks|ni|na|ta|ga)?|i|se(?:l)|se|e(?:s|l|st|le|lt|ks|ni|na|ta|d)))|i(?:d|sse|s|st|le|l|lt|ks|ni|na|ta|ga||sid|(?:de(?:sse|s|st|le|lt|l|ni|na|ks|e|ta|ga)?))?)"
```

Comment:
wait what's up with the `ga||sid` (I have no idea what this was supposed to test)

Reply:
idk you wrote it